modules/axhal/src/platform/aarch64_common/psci.rs

```diff
@@ -4,10 +4,7 @@
 
 use core::arch::asm;
 
-const PSCI_CPU_ON: u32 = 0x8400_0003;
```

Comment:
PSCI codes should be the same on all platforms, remove them in the platform toml files, and define here:

```rust
/// PSCI v0.2 interface
const fn psci_0_2_fn(func: u32, is_64bit: bool) -> u32 {
    if is_64bit {
        0xc4000000 + func
    } else {
        0x84000000 + func
    }
}

const PSCI_0_2_FN_CPU_SUSPEND: u32 = psci_0_2_fn(1, false);
const PSCI_0_2_FN_CPU_OFF: u32 = psci_0_2_fn(2, false);
const PSCI_0_2_FN_CPU_ON: u32 = psci_0_2_fn(3, false);
const PSCI_0_2_FN_SYSTEM_OFF: u32 = psci_0_2_fn(8, false);
const PSCI_0_2_FN_SYSTEM_RESET: u32 = psci_0_2_fn(9, false);

const PSCI_0_2_FN64_CPU_SUSPEND: u32 = psci_0_2_fn(1, true);
const PSCI_0_2_FN64_CPU_ON: u32 = psci_0_2_fn(3, true);
```

Reply:
All right. I'll research the definition of psci address.

Reply:
You can found these definitions at `/usr/include/linux/psci.h`